<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="exitApp">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connectRadio">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnectRadio">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QComboBox" name="radioSettings_Options">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>0/80/2M</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>0/80/250K</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="radioSettings">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio Settings:</string>
    </property>
   </widget>
   <widget class="QLabel" name="connectionStatus">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Connection Status: </string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>239</y>
      <width>671</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="Layout_CameraView"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>135</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="layout_Modes">
     <item>
      <widget class="QPushButton" name="pushButton_hoverMode">
       <property name="text">
        <string>Follow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_TakeMeasurement">
       <property name="text">
        <string>Take Measurement</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_CameraOnlyMode">
       <property name="text">
        <string>Only Camera View</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>41</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="LayoutColorBars">
     <item>
      <widget class="QSlider" name="verticalSlider_hue">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>110</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Layout_TrackingColor"/>
   </widget>
   <widget class="QLabel" name="setColor">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Set Tracking Color</string>
    </property>
   </widget>
   <widget class="QLabel" name="Hue_Num">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>80</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="display_ConnectionStatus">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ActualValues">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Actual Values</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ParameterTable">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameter Table</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SafeLandingMode">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Safe Landing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Emergency Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_EnterSetPoint">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter SetPoint</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_enableTestMode">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>EnableTestMode</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_disableTestMode">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DisableTestMode</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFollowMeFle">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFollowMeFle"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
